### PR TITLE
Issue #20 Script viewer gender toggle - Added

### PR DESCRIPTION
**Pull request by O-Isaac**

### CODE REVIEW

File: packages/db/src/Component/Script.tsx [hunk 1]
```diff
@@ -626,6 +627,9 @@ function parseDialogueGender(
         female: splitLine(female).map((word) =>
             parseDialogueBasic(region, word, parserDialogueState)
         ),
+        both: splitLine(`${female}{gender}${male}`).map((word) =>
```
Comment (squaresmile) [inline packages/db/src/Component/Script.tsx:630]: I wouldn't do another parse like this when both male and female components are available.

Comment (O-Isaac) [reply packages/db/src/Component/Script.tsx:630]: Understandable, I will try to access the two components, so I won't have to create a new.

File: packages/db/src/Component/ScriptDialogueLine.tsx [hunk 2]
```diff
@@ -96,17 +97,54 @@ const DialogueBasic = (props: {
...
-            const femaleComponents = component.female.map((component) => (
-                <DialogueBasic component={component} />
-            ));
-            return <>{mergeElements(femaleComponents, "")}</>;
+
+            const femaleComponents = component.female;
+            const maleComponents = component.male;
+            const popover = <DialogueGenderPopover female={femaleComponents} male={maleComponents} />
+
+            return <>{mergeElements([popover], "")}</>;
```
Comment (squaresmile) [inline packages/db/src/Component/ScriptDialogueLine.tsx:147]: You can just return `<DialogueGenderPopover female={component.female} male={component.male} />` here

File: packages/db/src/Component/ScriptDialogueLine.tsx [hunk 3]
```diff
@@ -96,17 +97,54 @@ const DialogueBasic = (props: {
...
+}) => {
+    const { female, male } = props;
+
+    const femaleComponent = female.map((component, index) => (
+        <DialogueBasic component={component} />
+    ))
+
+    const maleComponent = male.map((component, index) => (
+        <DialogueBasic component={component} />
+    ))
```
Comment (squaresmile) [inline packages/db/src/Component/ScriptDialogueLine.tsx:113]: These two arrays probably need the key prop.

**Pull request merged**